### PR TITLE
Street art queries

### PR DESCRIPTION
**Pull request by Gallup93**
Add gems
Setup GraphQL
Add street art query test
Expose endpoint for returning all street art

```
Finished in 0.25958 seconds (files took 2.65 seconds to load)
12 examples, 0 failures

Coverage report generated for RSpec to /Users/michael/turing/4module/projects/street_art_backend/coverage. 181 / 195 LOC (92.82%) covered.
```

### CODE REVIEW

File: app/controllers/graphql_controller.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+class GraphqlController < ApplicationController
```
Comment (stellakunzang) [inline app/controllers/graphql_controller.rb:1]: VERY comforting to see GraphQL in the rails universe with a controller!

File: app/graphql/types/mutation_type.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module Types
+  class MutationType < Types::BaseObject
+    # TODO: remove me
+    field :test_field, String, null: false,
+      description: "An example field added by the generator"
+    def test_field
+      "Hello World"
+    end
```
Comment (stellakunzang) [inline app/graphql/types/mutation_type.rb:8]: It's nice they include an example to guide us!

File: app/graphql/types/query_type.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    def users
+      User.all
+    end
+
+    #street art
+    field :street_arts, [Types::StreetArtType], null: false
+
+    def street_arts
+      StreetArt.all
+    end
```
Comment (stellakunzang) [inline app/graphql/types/query_type.rb:18]: woohoo!

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,9 @@
...
 #   Character.create(name: 'Luke', movie: movies.first)
+
+user = User.create(username: "Hank Hill", email: "ProPAIN@aol.com")
+
+user.street_arts.create(latitude: "39.7666", longitude: "104.9837", address: "RiNo", city: "Denver", state: "CO", zipcode: "80216", image_urls: ["https://cdn.5280.com/2017/04/RiNo-neighborhood-street-art.jpg"], description: "neat-o!", artist_name: "Larry", art_name: "cool-art-1", favorite: true, visited: true)
+user.street_arts.create(latitude: "39.7527", longitude: "105.0017", address: "LoDo", city: "Denver", state: "CO", zipcode: "80202", image_urls: ["https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTW8zd6DEZ5UHfY8r8NJB-jEcpGAlbHlwg6Cg&usqp=CAU"], description: "neat-o!", artist_name: "Larry",  art_name: "cool-art-2", instagram_handle: "@insta", favorite: false, visited: true)
+user.street_arts.create(latitude: "39.7581", longitude: "104.9874", address: "Five Points", city: "Denver", state: "CO", zipcode: "80216", image_urls: ["https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRe-qGNr3_dvdgNqE6g81qa4OTlSvmfbiGCRw&usqp=CAU"], description: "woww!", artist_name: "Sammi", instagram_handle: "@insta", favorite: true, visited: true)
```
Comment (stellakunzang) [inline db/seeds.rb:13]: This will be replaced with factory bot

File: spec/requests/art_requests_spec.rb [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    user.street_arts.create(latitude: "39.7527", longitude: "105.0017", address: "LoDo", city: "Denver", state: "CO", zipcode: "80202", image_urls: ["https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTW8zd6DEZ5UHfY8r8NJB-jEcpGAlbHlwg6Cg&usqp=CAU"], description: "neat-o!", artist_name: "Larry",  art_name: "cool-art-2", instagram_handle: "@insta", favorite: false, visited: true)
+    user.street_arts.create(latitude: "39.7581", longitude: "104.9874", address: "Five Points", city: "Denver", state: "CO", zipcode: "80216", image_urls: ["https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRe-qGNr3_dvdgNqE6g81qa4OTlSvmfbiGCRw&usqp=CAU"], description: "neat-o!", artist_name: "Larry", instagram_handle: "@insta", favorite: true, visited: true)
+
+    post "/graphql", params: {query: "{streetArts {id, latitude, longitude, address, city, state, zipcode, imageUrls, description, artistName, artName, instagramHandle, favorite, visited, createdAt, updatedAt, userId}}", operationName: nil, context: nil}
```
Comment (stellakunzang) [inline spec/requests/art_requests_spec.rb:14]: This too can be replaced with factories

**Review by stellakunzang — APPROVED** [review]

File: app/controllers/graphql_controller.rb (hunk 1 above)
Comment (elguapogordo) [reply app/controllers/graphql_controller.rb:1]: i agree

**Pull request merged**